dataservice/api/family/models.py

```diff
@@ -18,3 +20,10 @@ class Family(db.Model, Base):
     external_id = db.Column(db.Text(), doc='ID used by external study')
 
     participants = db.relationship(Participant, backref='family')
+
```

Comment:
If we change the above `participants` relation to have `backref=backref("family", cascade="all,delete")`, will that accomplish the same thing?

Reply:
`cascade = "all, delete"` is actually already doing the delete since the `all` translates to `save-update, merge, refresh-expire, expunge, delete`.

Maybe what you meant was `cascade = "all, delete-orphan"` which unfortunately does not work. It will delete the family when any one participant is deleted and then NULL out the other participant's foreign keys which refer to the deleted family.